onecodex/dataframes.py

```diff
@@ -135,9 +134,7 @@ def __init__(
         self.ocx_metadata = ocx_metadata
         self.ocx_normalized = ocx_normalized
 
-        super().__init__(
-            data=data, index=index, dtype=dtype, name=name, copy=copy, fastpath=fastpath
-        )
+        super().__init__(data=data, index=index, dtype=dtype, name=name, copy=copy)
```

Comment:
What do you think about this?

```
def __init__(self, ocx_rank=, ..., **kwargs):
   # handle ocx-specific args
   self.ocx_ = ...
   super().__init__(**kwargs)
```

For: maintain backwards compatibility, stays current with Pandas. Against: it's now possible to pass more kwargs to `pd.Series()`